src/components/RoomView/SetupGame.tsx

```diff
@@ -89,27 +84,31 @@ const SetupGame = () => {
   if (isLoading) return <p>Loading...</p>;
   else
     return (
+     <>
+     <div className="setUpContainer">
+      {showStartGame && 
```

Comment:
i tried this but it wasn't setting the cards when i did it this way. i think there might be a race condition involved but i don't have time to keep testing atm

Reply:
So - I might be wrong on this but - there's currently a bug with this component that I think has to do with how we are rendering it like this.
(please ignore that im using the version with the mui button)

I lost, and when I go to reset the game (this component shows again), it looks like this:
<img width="342" alt="Screen Shot 2023-02-10 at 4 45 44 PM" src="https://user-images.githubusercontent.com/71895647/218205555-f9531b77-3c12-4748-8cb9-20e0aeaf5927.png">
<img width="346" alt="Screen Shot 2023-02-10 at 4 45 39 PM" src="https://user-images.githubusercontent.com/71895647/218205564-45ea2647-17dd-4995-bad6-52e313baf80f.png">

Taking a quick look, when I clicked START on the first one (with nothing selected) it created a board with words only from the duet pack.

I think this bug is because now, the useState `selectedWordPackIds` doesn't get reset anymore because this would only happen on a component remount - and because we are conditionally rendering it here, this does not count as a remount.

Another option would be, (if conditionally rendering it in above comment does not work) after submission, we `setSelectedWordPackIds([])`